tests/frontend/test_ops_gate.py

```diff
@@ -172,7 +172,7 @@ def test_merge_measured_pars():
         ops.MeasureX | q[0]
         mpar = q[0].par  # measured parameter
         D = ops.Dgate(mpar)
-        F = ops.Dgate(1.0)
+        F = ops.Dgate(1)
```

Comment:
What was the issue that required a change here?

Reply:
a few lines below there's the assert: `assert F.merge(D).p[0] == mpar + 1`, which fails if the argument of Dgate is a float (+1 <-- int). Either we change one or the other.

Reply:
- Was this test working before? What changed so that line now causes failing tests?
- Since the natural argument to `Dgate` is a float, wouldn't we want to test that case? (i.e., changing to int here might mask a bug which a user would experience during natural use)

Reply:
I have changed the argument of the Dgate to a float, and the others accordingly. I don't know if they were passing before. They pass now.